index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/dist/css/styles.css">
+    <title>ChitChat - Login</title>
+  </head>
+  <body>
+    <main class="container">
+      <div class="d-flex justify-content-center mt-5">
+        <h1>Login or Create Your Account</h1>
+      </div>
+      <div class="row justify-content-center mt-5">
+        <div class="col-6">
+          <ul class="nav nav-tabs justify-content-center" role="tablist">
+            <li class="nav-item">
+              <button class="nav-link active" id="login-tab" data-toggle="tab" data-target="#login" type="button" aria-controls="login" aria-selected="true">Login</button>
+            </li>
+            <li class="nav-item">
+              <button class="nav-link" id="register-tab" data-toggle="tab" data-target="#register" type="button" aria-controls="register" aria-selected="false">Register</button>
+            </li>
+          </ul>
+
+          <div class="tab-content" id="myTabContent">
```

Comment:
It is not a good design to have both the sign-in and sign-up forms same time on a page. You could remove one. See an example at